### DISCUSSION TITLE
"XXXX  can't be processed. The connection is already closed."

### DISCUSSION THREAD

**Discussion by blackmad** [mmkal/handy-redis#305] [Ideas]
I've noticed a bunch of these errors, would it be helpful if all the handy-redis wrappers checked .nodeClient.connected before attempting commands?

cf https://github.com/NodeRedis/node-redis/issues/1157

**Comment by mmkal** [comment]
What would you suggest this library do here? In general, I would prefer to keep this is as a thin wrapper but if you can propose something specific that doesn't add a lot of responsibility to this package, I'd consider it.
